### DISCUSSION TITLE
`Flow is not contained in this Storage` when executing Prefect examples

### DISCUSSION THREAD

**Discussion by igonro** [PrefectHQ/prefect#5215] [Q&A] [answered]
I was following the documentation tutorial, and when I tried to execute the examples, I got this error:
```
Failed to load and execute Flow's environment: ValueError('Flow is not contained in this Storage')
```

I executed:
```
prefect create project "Prefect Examples"
prefect register --json https://docs.prefect.io/examples.json --project "Prefect Examples"
prefect agent local start -l prefect-examples
```

Then I just press the "Quick run" button in the "Example: Parameters" flow. Am I doing something wrong?

**Comment by kvnkho** [comment, marked as answer]
Thank you for flagging this @igonro . I can replicate and you are not doing anything wrong. I will look into this.

**Reply by kvnkho** [reply]
I made a PR [here](https://github.com/PrefectHQ/prefect/pull/5220) to fix this. This will probably get published next release though. For now, you can see the source code [here](https://github.com/PrefectHQ/prefect/tree/master/examples) and replace `flow.run()` with `flow.register("project_name")` in order to register these flows.

**Reply by igonro** [reply]
Thanks!
